fluent-react/src/useTranslate.ts

```diff
@@ -0,0 +1,28 @@
+import { useContext } from "react";
+import { FluentContext } from "./context";
+import { ReactLocalization } from "./localization";
+
+// Implementation of null object pattern for ReactLocalization
+const fallbackL10n: ReactLocalization = {
```

Comment:
Is this actually needed? I understand and agree with the intent, but it also seems to me that `useContext` is guaranteed to return an empty `ReactLocalization` thanks to how it's initialized here:

https://github.com/projectfluent/fluent.js/blob/c27994d75a8a708540a3b631617a271ec48253b9/fluent-react/src/context.ts#L4

Reply:
Okay, I removed that. I didn't notice that we are setting a default value on `createContext`